guides/source/getting_started.md

```diff
@@ -98,8 +98,9 @@ $ ruby --version
 ruby 2.7.0
 ```
 
-Rails requires Ruby version 2.7.0 or later. If the version number returned is
-less than that number (such as 2.3.7, or 1.8.7), you'll need to install a fresh copy of Ruby.
+Rails requires Ruby version 2.7.0 or later. It is preferred to use latest Ruby version.
```

Comment:
> It is preferred to use latest Ruby version.

Not always. Often times the very latest verion of Ruby has incompatibilities with particular gems.

Reply:
Yes I know but other rails docs suggested this.